code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -7,14 +7,12 @@
 	S["undershirt"]	>> pref.undershirt
 	S["socks"]		>> pref.socks
 	S["backbag"]	>> pref.backbag
-	S["gear"]		>> pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/save_character(var/savefile/S)
 	S["underwear"]	<< pref.underwear
 	S["undershirt"]	<< pref.undershirt
 	S["socks"]		<< pref.socks
 	S["backbag"]	<< pref.backbag
-	S["gear"]		<< pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/gather_load_query()
 	return list("ss13_characters" = list("vars" = list("underwear", "undershirt", "socks", "backbag", "gear"), "args" = list("id")))

```

Comment:
You should move gear out from the SQL system here, and into the loadout/loadout.dm. Procs you have to update:
* gather_load_query()
* gather_save_query()
* gather_save_parameters()
* sanitize_character().